test/e2e/indexTable/indexTable.spec.ts

```diff
@@ -24,26 +24,65 @@ test('IndexTable - bulk action', async ({ page }) => {
   const indexTablePage = new IndexTablePage(page);
   await indexTablePage.goto();
 
+  let isCheckboxCorrect: boolean, tempText: string | null;
+  isCheckboxCorrect = await indexTablePage.verifyCheckboxValues(
+    new Array(5).fill(false)
+  );
+  expect(isCheckboxCorrect).toBe(true);
+
   await indexTablePage.checkboxes.first().click();
   await expect(indexTablePage.bulkAction).toBeVisible();
-  await expect(indexTablePage.canvas).toHaveScreenshot(
-    'indextable-bulk-action-selected-all.png'
+
+  isCheckboxCorrect = await indexTablePage.verifyCheckboxValues(
+    new Array(5).fill(true)
   );
+  expect(isCheckboxCorrect).toBe(true);
+
+  tempText = await indexTablePage.tableHeadRow.textContent();
+  expect(tempText).toBe('4 selected candidates');
+
+  tempText = await indexTablePage.bulkAction.textContent();
+  expect(tempText).toContain('Reject');
+  expect(tempText).toContain('Move to');
 
   await indexTablePage.checkboxes.last().click();
 
   await expect(indexTablePage.bulkAction).toBeVisible();
 
-  await expect(indexTablePage.canvas).toHaveScreenshot(
-    'indextable-bulk-action-selected-row.png'
-  );
+  isCheckboxCorrect = await indexTablePage.verifyCheckboxValues([
+    false,
+    true,
+    true,
+    true,
+    true,
+  ]);
+  expect(isCheckboxCorrect).toBe(true);
+
+  tempText = await indexTablePage.tableHeadRow.textContent();
+  expect(tempText).toBe('3 selected candidates');
+
+  tempText = await indexTablePage.bulkAction.textContent();
+  expect(tempText).toContain('Reject');
+  expect(tempText).toContain('Move to');
 
   await indexTablePage.moveToAction.click();
   await expect(indexTablePage.actionList).toBeVisible();
 
-  await expect(indexTablePage.canvas).toHaveScreenshot(
-    'indextable-bulk-action-action-list.png'
-  );
+  isCheckboxCorrect = await indexTablePage.verifyCheckboxValues([
```

Comment:
i don't think you need lines 71-85 but verify there are 4 options from the bulk action list

Reply:
okay! Is verifying the 4 texts (assessment, interviewing, offered, hired) enough?

Reply:
I did this for now, let me know when you review it again if it is not sufficient